alf/examples/ddpg_fetchreach_conf.py

```diff
@@ -0,0 +1,92 @@
+# Copyright (c) 2022 Horizon Robotics and ALF Contributors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import functools
+
+import alf
+from alf.algorithms.agent import Agent
+from alf.algorithms.data_transformer import ObservationNormalizer
+from alf.algorithms.ddpg_algorithm import DdpgAlgorithm
+from alf.algorithms.one_step_loss import OneStepTDLoss
+from alf.environments import suite_robotics
+from alf.networks import ActorNetwork, CriticNetwork
+from alf.optimizers import AdamTF
+
+from alf.examples import ddpg_conf
+
+
+def define_config(name, default_value):
+    alf.define_config(name, default_value)
+    return alf.get_config_value('_CONFIG._USER.' + name)
+
+
+alf.config(
+    'create_environment',
+    env_load_fn=suite_robotics.load,
+    env_name='FetchReach-v1',
+    num_parallel_environments=38)
+
+alf.config('suite_robotics.SuccessWrapper', since_episode_steps=50)
+alf.config(
+    'suite_robotics.load',
+    use_success_wrapper=False,
+    gym_env_wrappers=(suite_robotics.SuccessWrapper, ))
+
+gamma = 0.98
+alf.config('OneStepTDLoss', gamma=gamma)
+alf.config('ReplayBuffer', gamma=gamma)
```

Comment:
If I understand it correctly this is specifically for HER which `keep_episodic_info`. Shall we move this, or add some documentation explaining why we set it here?

Reply:
Good point, added comment.